src/bsp/platform/cc3200/xi_bsp_hton.h

```diff
@@ -0,0 +1,2 @@
+#include <socket.h>
+typedef long int ssize_t;
```

Comment:
Ok, so this is the solution of one porting issue the cn-cbor has. cn-cbor needs hton function. And apparently it needs ssize_t too. hton is full platform dependent, architecture dependent? It is about the endianness. So to have cn-cbor on all of our target platforms we need hton which is platform dependent --> new BSP module without API (.h).

So this file is called "hton" but it isn't just hton but ssize_t too. And its actual job is to port cn-cbor. So it could be named "xi_bsp_cncbor_port.c". But it may help others who need hton. So I called it xi_bsp_hton.c but ssize_t appeared. So the best name could be: xi_bsp_hton_and_ssize_t.c :) Or xi_bsp_some_stuff.c :) Or we should break out from this BSP world representation and figure out the ultimate solution, or not ultimate but better than this.
So this is the tale of xi_bsp_hton.c